src/mlpack/methods/ann/layer/gru_impl.hpp

```diff
@@ -0,0 +1,307 @@
+/**
+ * @file lstm_impl.hpp
+ * @author Sumedh Ghaisas
+ *
+ * Implementation of the LSTM class, which implements a lstm network
+ * layer.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_GRU_IMPL_HPP
+#define MLPACK_METHODS_ANN_LAYER_GRU_IMPL_HPP
+
+// In case it hasn't yet been included.
+#include "linear.hpp"
```

Comment:
We should include ``gru.hpp`` here right, it looks like I did the same mistake for ``lstm_impl.hpp``.